File: tessoku/B02 - Divisor Check/B02.js
const main = (input) => {
    const [A, B] = input.trim().split(' ').map(Number);
    let result = 'No';
    for (let i = A; i <= B; i++) {
        if (100 % i == 0) {
            result = 'Yes';
            break;
        }
    }
    console.log(result);
}

main(require('fs').readFileSync('/dev/stdin', 'utf8'));

File: tessoku/B02 - Divisor Check/B02.cs
using System;

class Program
{
   static void Main()
   {
       string[] input = Console.ReadLine().Split(' ');
       int A = int.Parse(input[0]);
       int B = int.Parse(input[1]);

       string result = "No";
       for (int i = A; i <= B; i++)
       {
           if (100 % i == 0)
           {
               result = "Yes";
               break;
           }
       }
       Console.WriteLine(result);
   }
}